migrations/2022-04-12-114352_default_post_listing_type/up.sql

```diff
@@ -0,0 +1 @@
+alter table site add column default_post_listing_type varchar not null default 'All';
```

Comment:
Keep this like the theme, which is `text` not null. 

`default_theme                 | text                        |           | not null | 'browser'::text`

I still think `Local` is much better, because when you go to lemmy.ml for example, its full of posts from lemmygrad rn. We specifically changed it to Local from All a few weeks ago because of this problem.